### PR TITLE
Update mapping-1.16.2to1.17.json

### PR DESCRIPTION
**Pull request by FrostBird347**
Can't guarantee that all of the sounds match as well as they could, but it's far better than nothing.
#326

### CODE REVIEW

File: common/src/main/resources/assets/viabackwards/data/mapping-1.16.2to1.17.json [hunk 1]
```diff
@@ -1191,205 +1191,205 @@
...
+    "entity.axolotl.hurt": "entity.fox.hurt",
+    "entity.axolotl.idle_air": "entity.elder_guardian.ambient_land",
+    "entity.axolotl.idle_water": "block.bubble_column.bubble_pop",
+    "entity.axolotl.splash": "entity.generic.splash",
+    "entity.axolotl.swim": "entity.generic.swim",
+    "item.bucket.empty_axolotl": "item.bucket.empty",
+    "item.bucket.fill_axolotl": "item.bucket.fill",
+    "item.dye.use": "block.snow.fall",
+    "item.glow_ink_sac.use": "block.beehive.drip",
+    "entity.glow_squid.ambient": "entity.squid.swim",
```
Comment (kennytv) [inline common/src/main/resources/assets/viabackwards/data/mapping-1.16.2to1.17.json:1278]: The mapped sound doesn't seem to exist judging by the warning on startup

Comment (FrostBird347) [reply common/src/main/resources/assets/viabackwards/data/mapping-1.16.2to1.17.json:1278]: I understand how I made this mistake and will fix it.

**Comment by kennytv** [comment]
Now `block.beehive.ambient` doesn't exist :p

**Review by kennytv — APPROVED** [review]
Thank you, awesome

**Pull request merged**